src/server/frontend_xwayland/xwayland_wm.cpp

```diff
@@ -134,21 +136,28 @@ void mf::XWaylandWM::start()
     wm_get_resources();
     setup_visual_and_colormap();
 
-    values[0] =
-        XCB_EVENT_MASK_SUBSTRUCTURE_NOTIFY | XCB_EVENT_MASK_SUBSTRUCTURE_REDIRECT | XCB_EVENT_MASK_PROPERTY_CHANGE;
-    xcb_change_window_attributes(xcb_connection, xcb_screen->root, XCB_CW_EVENT_MASK, values);
+    std::vector<uint32_t> const attrib_values{
```

Comment:
Is there a particular reason that you're using a single-element `Vec<u32>` here, rather than just `uint32_t const attrib_value = XCB_EVENT_MASK_SUBSTRUCTURE_NOTIFY | XCB_EVENT_MASK_SUBSTRUCTURE_REDIRECT | XCB_EVENT_MASK_PROPERTY_CHANGE` (and then passing `&attrib_value` rather than `attrib_values.data()`)?

Reply:
Personally, I think "construction == initialization" is sufficient here:
```c++
    uint32_t const values[] = {
        XCB_EVENT_MASK_SUBSTRUCTURE_NOTIFY | XCB_EVENT_MASK_SUBSTRUCTURE_REDIRECT | XCB_EVENT_MASK_PROPERTY_CHANGE
        };
```

Reply:
There are probably lighter alternatives to `std::vector`, but I want to make it obvious to anyone reading that more elements could be added